src/event/window.rs

```diff
@@ -286,8 +286,8 @@ pub struct WindowMouseButtonEvent {
 	/// The new state of the mouse button.
 	pub state: ElementState,
 
-	/// The position of the mouse cursor inside the window.
-	pub position: PhysicalPosition<f64>,
+	/// The previous and current position of the mouse cursor inside the window.
+	pub positions: Option<(PhysicalPosition<f64>, PhysicalPosition<f64>)>,
```

Comment:
This should not change, as that would be break the public API . It's also much nicer to report them as two separate fields here.

Also, I did not make it an `Option` on purpose. It should not be possible to have mouse click events without cursor position. We may not have the needed information in the cache, but then that's a bug with us or with winit. We shouldn't complicate the public API for that.

Reply:
Although I think we should only report the previous position for mouse move events. Otherwise, how do we define "previous"?

Reply:
Yeah, I think "previous" would be set/changed only during mouse move events. Further previous should not be an Option as it is similar to current position, as in it always has a value?
If I am to not break the public API, I shouldnt change the name also right?